crates/proto-common/proto/sedachain/batching/v1/batching.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+package sedachain.batching.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/sedaprotocol/seda-chain/x/batching/types";
+
+// Batch is an aggregation of data request results along with validator
+// signatures used to prove these results on destination chains.
+message Batch {
+  // batch_number is a unique identifier of the batch incremented
+  // every time a batch is created.
+  uint64 batch_number = 1;
+  // block_height is the height at which the batch was created.
+  int64 block_height = 2;
+  // data_result_root is the hex-encoded root of the data result
+  // merkle tree.
+  string data_result_root = 3;
+  // validator_root is the hex-encoded root of the validator merkle
+  // tree.
+  string validator_root = 4;
+  // batch_id is the Keccack-256 hash of the batch content.
+  bytes batch_id = 5;
+  // proving_medatada is a field for additional proving data.
+  bytes proving_medatada = 6;
+}
+
+// TreeEntries are the given batch's data result tree entries and
+// validator tree entries.
+message TreeEntries {
+  // batch_number is the identifier of the batch the tree entries from.
+  uint64 batch_number = 1;
+  // data_result_entries are the entries (unhashed leaf contents) of
+  // the data result tree.
+  repeated bytes data_result_entries = 2;
+  // validator_entries are the entries (unhashed leaf contents) of
+  // the validator tree.
+  repeated bytes validator_entries = 3;
+}
+
+// BatchSignatures contains basic validator data and its batch signatures
+// under various cryptographic schemes.
+message BatchSignatures {
+  string validator_addr = 1
+      [ (cosmos_proto.scalar) = "cosmos.ValidatorAddressString" ];
+  int64 voting_power = 2;
```

Comment:
This is being removed in `hy/batch-skipping`

Reply:
I'll wait for that to merge on chain side before merging this then ^^